lib/ssh.py

```diff
@@ -23,7 +27,7 @@ class SSH():
     :type private: str
     """
 
-    def __init__(self, host, user, private_key):
+    def __init__(self, host: str, user: str, private_key):
```

Comment:
missing typing annotation for `private_key`

Reply:
@stephenlink1993 what's the best way to include type annotations for complex types that are defined by third party libraries?

Reply:
@jawang35 oh is `private_key` not type `str` in this function parameter?

Reply:
@jawang35 and good question! typically i search to see if there is a type annotation library for the 3rd party library. as an example, i used the library:
https://pypi.org/project/boto3-type-annotations/
for boto3 in `check-model-ready`. 
If there is not a typing annotation library, you can actually do something like this:
```
def __init__(private_key: RSAKey):
    self.private_key = private_key
```
^ `mypy` will complain about this, but i think its fine in this case 👍